### DISCUSSION TITLE
How to refine search results to documents authored by you.

### DISCUSSION THREAD

**Discussion by jlemoo** [microsoft-search/pnp-modern-search#3428] [Q&A] [answered]
I want people to be able to search on a "My Documents" view of a SharePoint list. "My Documents" would be just the ones they themselves authored. I was thinking I'd use the AuthorOWSUSER property, but how do I get my own email as the thing to filter to? It works if I hard-code it in like AuthorOWSUSER:jlemay@companyname.com

**Comment by kasperbolarsen** [comment]
It sounds like a case where the User token might be useful, https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/

**Reply by jlemoo** [reply]
Yeah, that could work. I'll see if I can figure out how to use. I'm not sure how to add a site column to a modern page.

**Reply by kasperbolarsen** [reply, marked as answer]
I am not sure I understand that, the setup could be as simple as 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/abcb0ee7-dd5d-4a8c-8e54-47c064e67fd1)

**Reply by jlemoo** [reply]
Thanks, that worked. I see it now under User Tokens about 2/3 down the page.
